install/infra/modules/aks/kubernetes.tf

```diff
@@ -59,18 +63,29 @@ resource "azurerm_kubernetes_cluster" "k8s" {
   }
 }
 
-resource "azurerm_kubernetes_cluster_node_pool" "pools" {
-  count = length(local.nodes) - 1
+resource "azurerm_kubernetes_cluster_node_pool" "regularws" {
+  kubernetes_cluster_id = azurerm_kubernetes_cluster.k8s.id
+  name                  = "regularws"
+  vm_size               = var.workspaces_machine_type
+
+  enable_auto_scaling  = true
+  min_count            = 1
+  max_count            = var.max_node_count_regular_workspaces
+  orchestrator_version = var.cluster_version
+  node_labels          = { "gitpod.io/workload_workspace_regular" = true }
+  vnet_subnet_id       = azurerm_subnet.network.id
+}
 
+resource "azurerm_kubernetes_cluster_node_pool" "headlessws" {
   kubernetes_cluster_id = azurerm_kubernetes_cluster.k8s.id
-  name                  = local.nodes[count.index + 1].name
-  vm_size               = local.machine
+  name                  = "headlessws"
```

Comment:
So would this mean that if we create multiple Azure setups, the nodepool name would conflict?

Reply:
The naming is unique to the cluster; multiple clusters in the same resource group or in separate resource groups should be fine. 👍